lib/ansible/modules/copy.py

```diff
@@ -75,7 +75,7 @@
     - As of Ansible 2.3, the mode may also be the special string C(preserve).
     - C(preserve) means that the file will be given the same permissions as the source file.
     - When doing a recursive copy, see also C(directory_mode).
-    type: path
+    type: str
```

Comment:
Mode is actually `type='raw'`

https://github.com/ansible/ansible/blob/f367cba1ab202ca43ee0b140e042dc00a41c144d/lib/ansible/module_utils/basic.py#L244

```suggestion
    type: raw
```